configure

```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69 for collectd 5.5.2.git.
+# Generated by GNU Autoconf 2.69 for collectd 5.5.2.1437.ga23b6f6+.
```

Comment:
Sorry, I was wrong about adding the generated files -- we've only done this a couple of times. I don't mind keeping these here if we could get rid of the superficial differences in the version, but that's probably more trouble than it's worth, so let's just back those out.